src/components/CTable.vue

```diff
@@ -70,18 +105,46 @@ export default {
           .includes(this.filter.toLowerCase())
       );
     }
+  },
+  methods: {
+    sortColumn(columnName, isRefresh = false) {
```

Comment:
I think you can use the previous definition I gave you after you parse the dates.

Reply:
> if you are sorting objects where the keys may be undefined this automatic sorting doesn’t happen and the results become inconsistent.
that's why Implemented a different version that adds undefined at the end. @antoinerabany